lib/project_types/extension/models/specification_handlers/default.rb

```diff
@@ -42,38 +42,22 @@ def valid_extension_contexts
           []
         end
 
-        def choose_port?(context)
-          cli_compatibility(context).accepts_port?
+        def choose_port?(_context)
+          false
         end
 
-        def establish_tunnel?(context)
-          cli_compatibility(context).accepts_tunnel_url?
+        def establish_tunnel?(_context)
+          false
```

Comment:
defaulting to false for both these options, other specification handlers can define this behavior